src/scenes/ApplicationRouter/portal/Review/index.tsx

```diff
@@ -9,6 +13,50 @@ import strings from 'src/strings';
 
 import ReviewCard from './ReviewCard';
 
+type ApplicationStatusProps = {
+  buttonLabel: string;
+  feedback?: string;
+  isFailure: boolean;
+  onClickButton: () => void;
+  title: string;
+};
+
+const ApplicationStatus = ({ buttonLabel, feedback, isFailure, onClickButton, title }: ApplicationStatusProps) => {
+  const { activeLocale } = useLocalization();
+  const theme = useTheme();
+
+  return !activeLocale ? undefined : (
+    <Card
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        flexGrow: 1,
+        alignItems: 'center',
+        padding: theme.spacing(8),
+      }}
+    >
+      <Box alignItems={'center'}>
+        <img src={isFailure ? '/assets/application-failure-splash.svg' : '/assets/application-success-splash.svg'} />
+      </Box>
+      <h3>{title}</h3>
+      {feedback && <Box dangerouslySetInnerHTML={{ __html: feedback }} justifyContent={'center'} />}
+      <Button label={buttonLabel} onClick={onClickButton} priority='secondary' />
+    </Card>
+  );
+};
+
+const ApplicationStatusSubmitted = () => (
+  <ApplicationStatus
+    buttonLabel={strings.EXIT_APPLICATION}
+    feedback='<p>Your Application has been submitted to the Accelerator team. We will review and score your answers.</p><p>Check back to see updates? Look for an email? [need copy]</p>'
+    isFailure={false}
+    onClickButton={() => {
+      // TODO: what does "Exit Application" do?
```

Comment:
Exit application just brings us back to home page. I have been using goToHome from useNavigateTo